src/features/defi/components/Deposit/PairDepositWithAllocation.tsx

```diff
@@ -0,0 +1,333 @@
+import { Button, Stack, useColorModeValue } from '@chakra-ui/react'
```

Comment:
This feels like this is duplicating a lot of view layer / logic when we could simply compose `<PairDeposit />` with children

Reply:
Agreed. In the interest of time, let's get this through as it is and then refactor the component in a separate PR.